SPEC.md

```diff
@@ -105,12 +105,23 @@ The test index makes use of symbolic language in describing connection and messa
 
 ### ZG-CONFORMANCE-004
     
-    The node responds with mtLEDGER_DATA for mtGET_LEDGER.
+    The node responds with mtLEDGER_DATA for mtGET_LEDGER with iType == LiBase.
```

Comment:
It might be worth concisely documenting what `iType` and `LiBase` mean here?

Reply:
As per slack: agreed to elaborate on this in the near future.